src/components/user-list-item.component.js

```diff
@@ -111,52 +124,44 @@ class UserListItemComponent extends Component {
             user.type === 'User' ? { user } : { organization: user }
           )}
         underlayColor={colors.greyLight}
-        style={!noBorderBottom && styles.borderContainer}
+        noBorderBottom={!noBorderBottom}
```

Comment:
Here is wired.

Reply:
hmm what do you mean by _wired_?

Reply:
@alejandronanez The prop name should be something like `hasBorderBottom`, because the value is `!noBorderBottom`.